tests/fixtures/help/update.txt

```diff
@@ -3,6 +3,13 @@ usage: obal update [-h] [-v] [-e EXTRA_VARS] package [package ...]
 Update a package to a newer version
 
 The newer version is either taken from an upstream repository or can be given via command line options.
+To update a package to a new version from the command line:
```

Comment:
Above this is a newline missing

Reply:
I don't get it though, I literally copied this from the output of `--help` and pasted it in here. That implies to me the testing is not verifying what a user sees?

Reply:
I swear I aint crazy, check this screenshot:

![screenshot from 2018-10-10 08-48-07](https://user-images.githubusercontent.com/293363/46737397-60a72800-cc69-11e8-937b-9d53488ebdc4.png)